cloud/src/autogluon/cloud/scripts/image_serve.py

```diff
@@ -0,0 +1,62 @@
+from autogluon.core.constants import REGRESSION
+from autogluon.core.utils import get_pred_from_proba_df
+from autogluon.vision import ImagePredictor
+from io import BytesIO
+from PIL import Image
+
+import pandas as pd
+import numpy as np
+
+
+def model_fn(model_dir):
+    """loads model from previously saved artifact"""
+    model = ImagePredictor.load(model_dir)
+
+    return model
```

Comment:
Is there `.persist()` call for image predictors?

Reply:
I don't think so. Why?

Reply:
`TabularPredictor` pre-loads models only on persist call this improves latency during inference times. I was not sure if `ImagePredictor` has a similar method or is it always in-memory.

Reply:
I see. For `TabularPredictor`, should we call `.persist()` after loading then?